rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -2283,6 +2291,7 @@ message GetUpgradeJournalResponse {
   Governance.Version deployed_version = 5;
 
   UpgradeJournal upgrade_journal = 4;
+  optional uint64 total_upgrade_journal_entries = 6;
```

Comment:
Doesn't seem like "total" is helping here. chatgpt seems to agree that unless cardinality of **_subsets_** is also of interest, "total" is superfluous.

Also, I see that we have many field [names of the form ${NOUN}_count][count]. Count is also something well-known in SQL. Our choice of NOUN is sometimes singular, and in other cases plural 🙄 I think singular is more grammatical, and chatgpt seems to agree. Therefore, it seems like upgrade_journal_entry_count would be the peak naming.

[count]: https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/dfinity/ic%24+f:%5C.did+f:nns%7Csns%7Cnervous+_count+-f:test&patternType=regexp&case=yes&sm=0

Reply:
Done!